MojioSDK/mojiosdksrc/src/main/java/io/moj/mobile/android/sdk/TimeFormatHelpers.java

```diff
@@ -59,6 +80,20 @@ public static DateTime fromServerFormatted(String date) {
         return dateTime;
     }
 
+    private static String getPaddedMilliseconds(String milliseconds) {
+        milliseconds = milliseconds == null ? "" : milliseconds;
+        if (milliseconds.length() == MILLISECOND_PRECISION)
+            return milliseconds;
+        else if (milliseconds.length() > MILLISECOND_PRECISION)
```

Comment:
I prefer to use braces once we add an else clause for clarity - in this scenario it doesn't actually affect total line count either